### PR TITLE
add katello 3.18 el8 config

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: configs/katello/3.18.yaml [hunk 1]
```diff
@@ -64,3 +71,25 @@
       - centos-sclo-rh-rhel-7
       - centos-7-server-updates
       - centos-7-server
+  - name: katello-3.18-el8
+    based_off: katello-nightly-el8
```
Comment (ekohl) [inline configs/katello/3.18.yaml:75]: There were already changes merged to nightly. How are we going to sync up the koji tags and git repo?

Comment (evgeni) [reply configs/katello/3.18.yaml:75]: Luckily, the changes are minimal:

```
% git diff --stat rpm/2.3..rpm/develop packages/katello/ 
 packages/katello/katello-repos/katello-repos.spec                           |  8 ++++----
 packages/katello/katello/katello.spec                                       |  8 ++++----
 packages/katello/rubygem-hammer_cli_katello/hammer_cli_katello-0.24.0.gem   |  1 -
 packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec |  8 +++++---
 packages/katello/rubygem-katello/katello-3.18.0.rc1.gem                     |  1 -
 packages/katello/rubygem-katello/rubygem-katello.spec                       | 21 +++++++++++----------
 6 files changed, 24 insertions(+), 23 deletions(-)
```

I'll just untag those packages and rebuild them from what's in 2.3.

**Review by ekohl — APPROVED** [review]

**Comment by evgeni** [comment]
```
% koji untag-build katello-3.18-el8 --all tfm-rubygem-hammer_cli_katello tfm-rubygem-katello katello katello-repos
```

**Comment by evgeni** [comment]
manually built hammer: http://koji.katello.org/koji/taskinfo?taskID=384546

the rest will be built anyways when we do rc2

**Pull request merged**